src/raster/r.accumulate/r.accumulate.html

```diff
@@ -288,6 +288,7 @@ <h3>Longest flow path calculation and subwatershed delineation in one run</h3>
 <h2>SEE ALSO</h2>
 
 <em>
+<a href="https://grass.osgeo.org/grass-stable/manuals/addons/r.flowaccumulation.html">r.flowaccumulation</a>,
```

Comment:
```suggestion
<a href="r.flowaccumulation.html">r.flowaccumulation</a>,
```

For addons, please just use the relative path.